serveradmin/serverdb/query_executer.py

```diff
@@ -258,6 +258,6 @@ def _get_servers(filters, attribute_lookup, related_vias):
     # easy: get and execute the raw SQL query.
     sql_query = get_server_query(attribute_filters, related_vias)
     try:
-        return list(Server.objects.raw(sql_query))
+        return list(Server.objects.defer('intern_ip').raw(sql_query))
```

Comment:
actually in this case it only defers the ip object creation which saves a lot of time because django is not parsing them. this is because the intern_ip is already inside `sql_query` and will always be selected. see my comment below for why i do not want to touch this query logic now :)

Reply:
Ack!